stages/org.osbuild.dracut

```diff
@@ -201,15 +203,35 @@ def main(tree, options):
 
     opts += extra
 
-    for kver in kernels:
-        print(f"Building initramfs for {kver}", file=sys.stderr)
-
-        subprocess.run(["/usr/sbin/chroot", tree,
-                        "/usr/bin/dracut",
-                        "--no-hostonly",
-                        "--kver", kver]
-                       + opts,
-                       check=True)
+    mounts = [
+        {"path": "/dev", "bind": True, "ro": False, "mode": 0o755},
+        {"path": "/proc", "bind": True, "ro": True, "mode": 0o555},
+        {"path": "/sys", "bind": True, "ro": True, "mode": 0o555},
+    ]
+    with MountGuard() as mounter:
+        for mount in mounts:
+            ro = mount["ro"]
+            bind = mount["bind"]
+            src = mount["path"]
+            mode = mount["mode"]
+            dest = os.path.join(tree, src.lstrip("/"))
+            os.makedirs(dest, exist_ok=True)
+            os.chmod(dest, mount.get("mode", mode))
```

Comment:
(nitpick) doing this in two steps is racy (which of course is not a big deal here but I still wanted to mention it), could this be done via `os.makedirs(..., mode=mode)` ?

Reply:
That's how it should be in there :)

Reply:
Oh, I just was saying that you are right, that it should be `os.chmod(dest, mode)` instead of `os.chmod(dest, mount.get("mode", mode))`